### PR TITLE
Blended component

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Lo and behold, here is the blended component.

@JonNorman I had to change the way you deal with the soulmates URL, because config ends up being inserted into every component and the `SubFeed` variable only exists in the soulmates format, DFP complains about an inexistent variable in other components. Actually, I'm thinking this whole config object was a bad idea.

### CODE REVIEW

**Comment by JonNorman** [comment]
Ah I think part of that was covered here? https://github.com/guardian/commercial-templates/pull/45

File: src/_shared/js/utils.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (JonNorman) [inline src/_shared/js/utils.js]: Shall we move the `dev.js` methods here?

**Comment by JonNorman** [comment]
Looks good to me - but could you take a look at my PR #45 before merging? Happy to go with either method of changing the soulmates endpoint (I've got a frontend PR open for this too https://github.com/guardian/frontend/pull/14531)

**Comment by regiskuckaertz** [comment]
@JonNorman by Jove! Yours is better, please merge and I'll remove my changes.

File: src/_shared/js/utils.js (hunk 1 above)
Comment (regiskuckaertz) [inline src/_shared/js/utils.js]: Oh, I forgot about this one. Yeah slowly but surely the `js` and `scss` folders are getting unwieldy, let's think about this in a future comprehensive PR.

**Comment by JonNorman** [comment]
Okay great! Merged my changes :)

**Pull request merged**